libpod/container.go

```diff
@@ -51,6 +51,9 @@ const CgroupfsDefaultCgroupParent = "/libpod_parent"
 // manager in libpod
 const SystemdDefaultCgroupParent = "machine.slice"
 
+// DevShmPath is the global string const pointing to /dev/shm, because lint wanted it
+const DevShmPath = "/dev/shm"
```

Comment:
You don't need to change anything, but FYI, we nuked that linter, in a large part because of things like this.

Reply:
Thanks, I removed all the DevShmPath changes I did.